src/rpcgovernance.cpp

```diff
@@ -846,6 +846,7 @@ UniValue getgovernanceinfo(const UniValue& params, bool fHelp)
             "{\n"
             "  \"governanceminquorum\": xxxxx,           (numeric) the absolute minimum number of votes needed to trigger a governance action\n"
             "  \"masternodewatchdogmaxseconds\": xxxxx,  (numeric) sentinel watchdog expiration time in seconds\n"
+            "  \"hassentinelping\": xxxxx,               (boolean) has sentinel ping rpc call\n"
```

Comment:
ok, I wasn't quite accurate about `constants`, let's read that as `governance globals` instead. In any case, I think that node specific field like that doesn't belong here. It's also excessive since you can simply get `protocolversion` via `getinfo`.